src/wirecraft/client/game.py

```diff
@@ -1,22 +1,21 @@
+from __future__ import annotations
+
 import sys
-from collections.abc import Callable
-from dataclasses import dataclass
 from enum import Enum
+from typing import TYPE_CHECKING
 
 import pygame
 from pygame.locals import QUIT
 
-from .assets import CLOSE_BUTTON, INVENTORY_BUTTON, SWITCH_DEVICE
+from .constants import BLACK, FPS, GREY, PADDING, RES_LIST, WHITE
 from .server_interface import ServerInterface
+from .ui import Button, Cable, Device, Resolution, Window
+from .ui.assets import INVENTORY_BUTTON
+
+if TYPE_CHECKING:
+    from .ui import Camera
 
-BLACK = (0, 0, 0)
-WHITE = (255, 255, 255)
-GREY = (128, 128, 128)
-RED = (255, 0, 0)
-RES_LIST = [(3024, 1964), (1920, 1080), (1920, 1200)]
 FLAGS = pygame.FULLSCREEN | pygame.NOFRAME | pygame.SCALED
```

Comment:
Is there a particular reason not to include them in the constant file?

Reply:
Only because I didn't want to import pygame in constants.py
But, this should 100% be defined in constants